### PR TITLE
Update Account Settings page

### PR DESCRIPTION
**Pull request by bmanifold**
Why:

* The current account settings page shows the user profile info and billing information.  The billing system is not currently implemented and can be hidden for the time being.  As for the user profile information, it was decided that it would probably be better to send the user to the 'Actors' show page for the currently logged in user.

new-account-settings-page

### CODE REVIEW

File: elixir/apps/web/lib/web/live/settings/account.ex [hunk 1]
```diff
@@ -6,167 +6,23 @@ defmodule Web.Settings.Account do
...
-                <.link class="text-xs text-blue-600 dark:text-blue-500 hover:underline" href="#">
-                  Switch to monthly billing
-                </.link>
-              </p>
-            </td>
-          </tr>
-        </tbody>
-      </table>
+      <.vertical_table>
+        <.vertical_table_row>
```
Comment (AndrewDryga) [inline elixir/apps/web/lib/web/live/settings/account.ex:21]: @bmanifold I think we should render the account name too.

**Comment by AndrewDryga** [comment]
@bmanifold please add basic tests and LGTM.

File: elixir/apps/web/lib/web/live/settings/account.ex (hunk 1 above)
Comment (bmanifold) [reply elixir/apps/web/lib/web/live/settings/account.ex:21]: Yeah, that's probably a good idea.

**Comment by bmanifold** [comment]
Apologies for forgetting tests. With this page being so static tests slipped my mind.  I'll get some added.

File: elixir/apps/web/lib/web/live/settings/account.ex (hunk 1 above)
Comment (bmanifold) [reply elixir/apps/web/lib/web/live/settings/account.ex:21]: Account name has been added to the table

**Comment by bmanifold** [comment]
Here's a preview with the Account name added to the table

new-account-settings-page-update

**Review by AndrewDryga — APPROVED** [review]

**Pull request merged**